src/test/java/Entities/QuantityRecipeItemTest.java

```diff
@@ -0,0 +1,36 @@
+package Entities;
+
+import Entities.Implementations.IngredientImpl;
+import Entities.Implementations.VolumetricRecipeItem;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import java.util.Collections;
+
+public class QuantityRecipeItemTest {
+
+    @Test
+    public void testDisplayOneQuantity() {
+        IngredientImpl ingredient = new IngredientImpl("lemon", Collections.emptyList());
+        VolumetricRecipeItem quantiyitem = new VolumetricRecipeItem(ingredient, 1, false);
+
+        Assertions.assertEquals("1 lemon", quantiyitem.display());
+
+    }
+
+    @Test
+    public void testDisplaySmallQuantity() {
+        IngredientImpl ingredient = new IngredientImpl("lemon", Collections.emptyList());
+        VolumetricRecipeItem quantiyitem = new VolumetricRecipeItem(ingredient, 0.5f, false);
+
+        Assertions.assertEquals("0.5 of a lemon", quantiyitem.display());
+
+    }
+
+    @Test
+    public void testDisplayLargeQuantity() {
+        IngredientImpl ingredient = new IngredientImpl("lemon", Collections.emptyList());
```

Comment:
The convention we are using for this repo is that each public method gets one method, in this case `testDisplay` which should contain each of these sequentially.

Reply:
Also make sure we test all cases, such as correctly using `an` over `a`.

Reply:
do you want me to make a testdisplay class? and then have there be tests for both volume and quantity under the same class. @DerekCresswell

Reply:
No. One test class per class, one test method per method.